lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/MohamedTest.java

```diff
@@ -0,0 +1,60 @@
+package com.codedifferently.lesson7;
+
+// import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.MohamedObjects.Main.CPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.GPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.IllegalYearExeption;
+import com.codedifferently.lesson7.MohamedObjects.Main.cpuBrands;
+import com.codedifferently.lesson7.MohamedObjects.Main.portTypes;
+import java.util.*;
+import org.junit.jupiter.api.Test;
+
+class MohamedTest {
```

Comment:
Move to sub-folder named `MohamedObjects`. Also, the test file name should correspond to the object under test. You're testing multiple objects though, so maybe name `ComputerPartsTest`.

Reply:
On second thought, you should also prefer lowercase alphanumeric package name. See https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html for more guidance here.